### PR TITLE
Resolve dialyzer issues

### PR DESCRIPTION
**Pull request by alanvardy**

### CODE REVIEW

**Comment by alanvardy** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10** <a href="https://app.graphite.dev/github/pr/theblitzapp/elixir_app_gen/10" target="_blank">Graphite</a> 
    * **PR #11** <a href="https://app.graphite.dev/github/pr/theblitzapp/elixir_app_gen/11" target="_blank">Graphite</a> 
      * **PR #12** <a href="https://app.graphite.dev/github/pr/theblitzapp/elixir_app_gen/12" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/theblitzapp/elixir_app_gen/12?utm_source=stack-comment).

File: lib/app_gen/ecto_context_generator.ex [hunk 1]
```diff
@@ -65,7 +65,6 @@ defmodule AppGen.EctoContextGenerator do
     repo_opt =
       case repo_underscore_name(repo) do
         "repo" -> ", @repo"
-        nil -> ""
```
Comment (alanvardy) [inline lib/app_gen/ecto_context_generator.ex:68]: I don't think that this function can return nil

File: lib/app_gen/input_arguments/required.ex [hunk 2]
```diff
@@ -90,7 +90,7 @@ defmodule AppGen.InputArguments.Required do
          crud_options
        ) do
     case split_input_object(objects, ecto_schema) do
-      {nil, _} ->
+      {[], _} ->
```
Comment (alanvardy) [inline lib/app_gen/input_arguments/required.ex:93]: Is an empty list what you meant?

File: lib/mix/tasks/schema.ex [hunk 3]
```diff
@@ -39,7 +39,7 @@ defmodule Mix.Tasks.AppGen.Schema do
       validate_repo!(opts[:repo])
 
       with :ok <- Mix.Tasks.Phx.Gen.Schema.run(extra_args) do
-        require_new_schema_file(extra_args)
+        _ = require_new_schema_file(extra_args)
```
Comment (alanvardy) [inline lib/mix/tasks/schema.ex:42]: Ignore this :laughing:

**Review by MikaAK — APPROVED** [review]

**Pull request merged**